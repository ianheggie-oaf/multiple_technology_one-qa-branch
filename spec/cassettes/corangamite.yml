---
http_interactions:
- request:
    method: get
    uri: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationSearchResults.aspx?Field=S&Period=L28&f=$P1.ETR.SEARCH.SL28&r=CSC.WEBGUEST
    body:
      encoding: US-ASCII
      string: ''
    headers:
      Accept-Encoding:
      - gzip,deflate,identity
      Accept:
      - "*/*"
      User-Agent:
      - Mechanize/2.7.7 Ruby/2.5.8p224 (http://github.com/sparklemotion/mechanize/)
      Accept-Charset:
      - ISO-8859-1,utf-8;q=0.7,*;q=0.7
      Accept-Language:
      - en-us,en;q=0.5
      Host:
      - eservices.corangamite.vic.gov.au
      Connection:
      - keep-alive
      Keep-Alive:
      - '300'
  response:
    status:
      code: 200
      message: OK
    headers:
      Cache-Control:
      - no-cache, no-store
      Pragma:
      - no-cache
      Content-Type:
      - text/html; charset=utf-8
      Content-Encoding:
      - gzip
      Expires:
      - "-1"
      Vary:
      - Accept-Encoding
      Server:
      - ''
      Set-Cookie:
      - ASP.NET_SessionId=ye2pw255nlmdon45dg2kus55; path=/; HttpOnly
      X-Aspnet-Version:
      - 2.0.50727
      X-Powered-By:
      - ''
      Strict-Transport-Security:
      - max-age=31536000
      Date:
      - Wed, 19 Feb 2025 03:28:56 GMT
      Content-Length:
      - '8234'
    body:
      encoding: ASCII-8BIT
      string: !binary |-
        H4sIAAAAAAAEAO29B2AcSZYlJi9tynt/SvVK1+B0oQiAYBMk2JBAEOzBiM3mkuwdaUcjKasqgcplVmVdZhZAzO2dvPfee++999577733ujudTif33/8/XGZkAWz2zkrayZ4hgKrIHz9+fB8/In7j5DdOHv+uT788efP7vDxN5+2iTF9+9eT52Un60fbdu9+9d3L37tM3T9Pf+9tvvnie7o530tdtXUzbu3dPX3yUfjRv29Wju3evrq7GV/fGVX1x982ru+8AZRev6a/bDb8znrWzj464Q3ycvluUy+azCIzdhw8fyqto/nieZ7Ojx23Rljn9+WMnVZ0tL7JF0ebp63lR52n+Oq8vi2nepNvp8WpVFtOsLapl+jrP6uk8fZU367JtCNJdAfK4LJZv03mdn0vvDXV/Xi3bZnxRVRdlnq2KZjytFnenTfN7nFNP5fVnr6t1Pc0/eZ0tm09e1tVHaZ2Xn33UtNdl3szzvP0ovfu14H6xLotvDNhPZgQouxHceHyX/keU+v3fzPNF3tw9eX1y92l+nhGZxgTuo7S9XuWffdTm71qAjwO8y/NCVJ1Us2v8bKZ1sWr9d386u8zk04/Spp5+9tHdu9lPZ++6I8Bnd8ti0tz96V+0zuvru7vjg/E9/WP8081H1JmAOUp/4ySl5/F5VS/SZbagnrJmtczbZ/TBR+kib+fV7LOPVlVDXWZTsMFnH+Vv6mz6lsZrOEMYQ/liTADe/R7Pirycffb6F2aL1eHLvC4IyvO9A/7z/LPffW//5e749M2r8evT41cn3x6/Nt/VnxHtxt89ffL5V6ev33yUFtQ5wTMIgSyz4hI/iuVqbYgzL2azfPmRDuD3//1/8uz0u6/fHL85FQDBB5dZuaZGd69OXz69+ur3+uLN6f0vnl784Iunb3deXD18++ziO4unez/x3eOLs6f4+eT47FR/PsHP48Xv8/Knnvw+p8cXp/h58lMX+s7VAWG9f/qTn//k/uz4q++8+u6r1ezkyYvJ3qvy2xft08nebvv7fPf+2zeLZ+1P/cRX1O7ZD7J3+y9/Yu9hO/nus/VPnT67/ql7P9n+1Hfv7zx5tfqJL776yZ/46u2L3em9V5eT75Y/+H1+71eryd79Zz/1ebnz5rsP3/7Uq/3v/MR3X/z0T/3eL37weu8n79PfP/ETe892p3s/+YOfXDxrsu++ms8+Ly8ni4v85Nmr+e/ztlz/FOF2/BPHx9++mH012XvY/OTn5RW99zJo/3Z2Pbn3k1c/9d1Xb4kGkTEeHD/5id3XP/Xdn7qmvquz0xfz32dvXn776vTZE+pnutecH198lX/19Nmzr5595+ns936xQ2NfTfee7WTffbj+avGTi+niJ3+Q2fde0P9fVd+++urJ8fHx+ZPFV+dP9n7y7U8dE/2pv+P5/rd/clH+IPvud5qf+u6c8Dg7Ob44ePlTxwcvn500x8evfp/Pv33x6svf57u75ZOfWDXTnZ/anSxecF/fvnh3SnjOZ999t4Mxzr771bPT61c/8cXr+89+8tnZ+qudn3xCND57Xr744oufvsiflk/mU8zP7/2qnJY/NZ98+yfLn3ryE/XxT5yeHO+BHlPQ43PM/dN3v8/Jt6+On7UXZwc/8dP3f2Ly+bP1ZEE0vn7yEz/1e3+nzX7vn7j4au8nCR7R6Nl3yum9n2xm3/7i4PnO2Sdnxzu/18vT77x+ebHze73+7u9zkX/34e7Z5/P5bPHVBfHI/bNvP9sl2vIYptdPrn/quzOCQ/O/mF7MPp+XZ5//1OXk83eXsz3ubz799hOavxcyf8svLqafvyt/n9/7BbV7QX/T3N/7CXnv2ZOG4K+z795fnp3SGD9/dj37fLecLH/iYrK3f/HF0y92X7x5+4MXPzi9+uJk/+LNvVfVT/3eZxe/z97D9ezzZz9N/PyWaDPPvvvuBwR7PimeFD/1+smC+GX9UyfHBz/x+klFc7t4+fo71ezbr66mP6gun9+b3Ztd3//pyb3vzCeL2Zx4eeenXt/fy777xfqn9h7uPV88232++4R47kk5Xb66T79bHJ/vvtghvt+dff7wOn+969H4IcnT/Z3f57svdp5/97T96vN3BPt+OS3OPsk+/8nrn3pzdfkTb/YPfqI8++SnPv+Jk2OWa/pJfEP8c358/NUXdm5Owfc/+QNpZ7//fb746S8uv3h6evnFTx9fv3j9ZGdyffzuy9cHV18UB9dfPD3bRfsnP4H2P5GfnM6uSX6+yn/vJ8SD74i3vvPkJ3baN6++evb6J3Yu8lP5/ul0QWP/vb+z+n2+erea3vuJ6wc/QcJI/zt+eZeeA/C/Plcn9M/xV89+4t3z8pj+Pyt/n+L+y99nsSp/n3uvnvzU58+uCOfr6fXVxU8u6OcezftP77yjOdt5/tPH6y9Ori5+gvkNtNhd/9Tv/ZM7xNfNc8J4RnJEvPL8p37vknRAW05+endN8tE8OT6lHk+mxz/54j7pnHLy+j7R+B3JnOHD+98m3VTSGH76+Vvg/+rii59sJz/59HT9Ff3/J/d+snj+9mGRLX7yp2enz97+Pr/3E4z1B89JnwRjf004U/uN75w8+S5+ot+Xb06vnv/0T9CY9q/o86eE687s9/5O+ZJwmf3er65/n+9eNcTXu7/P4t3q99khb+QnX11mez+5pu93SUZmv/dPfEHjOv78996j/r+7+4MnP/nwB9nvvSpPnj3cM32A8N/ZvYnW979NY3j7E/e+s8Lf2XefTX7vnzijN49JIb19Sj9lLpmOx98+vqJ/j48BWv45/onyxnF3aSXwvyPv7zx9An45Pv4acL6i146/Pfm9X3z39/nuu92f+ol5O90V/fbFxf12uvPwitqRHDy8Jnv09qd+YkWfke74Nk3w2/Lt08XuD7769ncupzTu2bdL1rWkE5/Q/5lPn1ww/O+GMnP87oufrq6+eEr/P3ny5M1PkN6k59lPn95//oPj6+c/OLv64qe/uvjizemnpHPw/4uvTnc+f8J0+87pT1z8Pjz4p1+9u/ypz2fl5Ls/uZ6xrH71He70uCWZ/g7bamubL+gn2wzYqJLn6+zZq/uEt2dPDs6yzzttX737cnL9hPTrT11OF7vUB8lT8WRFupd08rMV6Z1i8vlXZD9/4vc5vvrJp8QXzRdPj98SrUgvEl4nr36a7Nxr0tm7k2+/+sEr4pGfOnlF9vHJ7/1T3/0O4/Gdz8kOPj2++s7ns+ufekY8JnP01e/z3e8Qnz67JhtXzq4ODn6vq+Pf6/gnLk6Ov/sTz37CvHNKc7EkmVz+xNvf595P/oBs6A8gB9MdwPiJt6Rzr8AL2u9Pzz6/Ipq/ejv79k/Z/t98Xv5g9uynVj/1e0/Zbj4he/ZTn78iGh6fPF38xLNvR/qafP6dxujJV58/I1764mX2+aviLDvPJ/vNwy9O8mb5nYN395rZd7IvHi6mx599BleWvLO74qXRb+aXH7udx/b56YvTV8dvvnzVc928b9SHe/Bw/8nu6e4z6XNzB6c/efrizU8ePz97evzm7EuKtQR672MFTe7hd59dPF/t/T5Pd48vfq979x9++/d6evX83sEnr3ee/MTzpzs/dffuyfHzdd0u758c/15Pf3r1g5z+vlv8oi+XT69+r7z4lH/u/nSGn8+n8vfzB/Lz98p++vfin3Np//yskL/v//Tz8+LpxfOLouafd+UnwXvAP3d/eoKfBE/+vntx8N0vTn7i9/rpxXfOX53MX9xf3T3+vWaffD6p3u0//EXl+u1O2b78oqyfCIlkWsTlx0Of0PSk0zJrKHCckFN/UVfr5Wx7SoFRVizzGh432lFj+/jvIGbJ6+1q3dK/wVs/Rk+kpW2T6gdldk1vb++l1JDCiPPinXk79npDUep2WV1UttGPPc40DDNRHSLfqQtqxxTKUoR0Oc7Wd7237mbmd0MV+tX/3f+VfjOILLPL4oKjno2jBntRUwr1qpL4iX7Vtz6KAHIg0lnWZtuLfLneVj6u6uIH+Jpjd4Iu4BXGRV1wCoA+Dj6fVuX2u2Z7d898+WOP16X5sjnnDkQEmun2grqWT0zjH6Po1rQmNLebnCO/7V+0zmoaMgEmNjgv6qbdJgIjcLzkX6jhL9redWDea26+XS1ynzjbZX6ZlyE4ixW9u9guiG6IZYujx80qWwbfIV7+6AgwKdalLz0gbubp97Jwf9xi0PiC4WOkjrr08nuM9IRkbFqUscGmHx3pt18bzT1GErH6Zb69ytp5+PdFzn8TSjKIe19zEJS0WeV1ez0wCvP1BwyjQ+39r4noSbVYrJfFIKb2+6+N6r0uqve/Jqovy4yybtsZaWBCj3JTAyhLu5TapfQZtfvmUP/0a6L+ZN2QAkNuy+tOMf7oyHz7tfHc7+L54GvieQJdOqVkltebRVO/TL8aRPTx3XWpv3v2Yeh38yv+owGYJ9TUy5ZmMGZLqGXw8GvQ2VDYxrDI7woMf3Qh0XfOrBKU3sNgFQDeAjYMCH/Qzzgy5gnf7lqdH6MWQRMGXCwJaAiemuKh1vbh18xwuTVhFg4VH9LP2wzyd93eTp9XF8Uy3d4eaoMOFXyzyqdFViLJOwQSz/A3eAKrxJ1vgmUe4maMetqWOzu/P/3LL/7+08WMf0EWlmS/eZtfE0zKLCMD/9lHx/xh+nvl149S+lTEwaWLH/3+v/+seklJ3Cfk4219TFB3dn43+pdB/m4G9sejjz++89ER/w4BIGQ2PjeMxRpe+n3wGfjWSA792nt4Lk9J9zGimE/60D79d3hemaRN8yQD79GL7EDSl72HyC/UM8okb3Q1ZEilvNl9+WpVV7O7380nlBRv7ubG+N19uXv383XetPBDJC8eJrh/4flnvxsy4S9fjfnvb39xOv7Js++Ka0bzAw5YzPD2C/IlBzDG87hYXMiSAK1E0P8ID7MYoQsRd88WZPubu6eLVXs9vijO/U4KfGe6SSdVTb72Zx/tWGmbm68QRVB3vQfs0v8mNovRz+gj0xVLMrw6+izWjJFuGqhq0gfHdZ7F6OI3fZM1b4kRybmUPtLPUnz8onpZF6RRIm/jiX8KuAMv4Bn+xjwAIGiQDiOf+yUhxq49fTf4bP4WTxfstzlcugmueURRMbmYHQzFfv96VX9B0J4XTQtVtLP7+5eTEp98TnHiynTy5ZL0ZJ6+zK6pazgjAHeLrmNzHHu82IU6kL6BEoYHz+HIyixxfk5o3D1eEkeX9Bu1b6OCh8UmuxD18vcZH784++L4+evx6Yuf+OhIXjcDAm+zDxDvyi2D0Z9of7v+Xr40nTkAt+vxaT5pq5p+Q9Nbdfb09AnlTbQ/ef12XZ0tz0lGLkiPLFv6Gz9u1eHZi2evzl58fio9+lBu1++rrM0b+gUtb9UhJYZOdXz87u26eS1Wnt+gv/HCrXp7/fL0BD1KhwolZTDslJ/Ms5p07Y1IvJ5XqxVR5oS83OCPG7F4efz7jE+OX71ha3FkXk3xru2NsjvirZIIbXweT+pBxW6ezd/i+SGpoL2oCvqKjHRKfEaLxCxJP7c6CNjcfUmNr8iS0rJ4fvOEwgX46vXpq5fHr1/LpNKC90XVkpEjNhJIdmaj/b3KLwifvL5VV69OP0dv4y9evCGJ0Tct/B8259xiDoa/jX/z9WAOvdX/hLmd2dU5GK/ykjTJ7DkFDRQHK7t8fV8Dj+uGWH/n939VXMxbdAe34fdvm1XYpfT4UasN2K9gCN4XRmL0zZRfNeMmLse/xAKWvfI3NcUMnpF6Tf7ZdE5cTU5lI8z2rMjL2Wevmcte5nVRzT57813+axMbvnk1fn16/Ork2+PXb4jfX9PabkH8PkvfzIsm/W6ev/UZkv79UJyevxdOzwOcnmdN+7OB05sv+K9b4vTmCx8nptMX5AjPv2Gknr8XUs8DpJhQPytIPeC/bosUJYg6SD1In2bXxOrfLFa7+/znbdHapTRmB6/d/Z8VxPYO+M/bIrZ30ENs7+AbQuwpd6uIvZdmeArN8DQn47SgEENZ/psRwwCp91INT6EaPKSYWD8LSL2XbngK3eAhxZT6huQwwOq9lMNTKAcPKybVzwpW76MdnkI7dLH6ptRDiNZ7qYen1LyP2DemH0LM3ks/PKXmfcwiCkL8xq7T8ayq6M2Pjn7hctKsDq23oT+H3Z/4N+/zKWPCDlTTfDsvV8fL2fGMhgDXiL6OPvKN+Vd+dh+DO/0aPEOfd59+Cx9TzW7FkORmSlrX8rtV/ZYinmkeewUPsUsI+021QpYtlrbDE/8Uz+P5roBij1Tcyd9/vvuSMohvkIwOMxrCjKly4+O789336vB9PnVP/Hv5VP7thptKlN9/um5ON8rTR2nTXiPjPiuaVZldPyo4+bU9Kavp28OrYtbOH+3u7Pzuh0TcJ1ioWy4RmJPQkWXtUePJEeHz+MkrosnZeXpdrdN5dplTDuE6/UXIG1PvTVrnF1k9A5R2nqfnVVlWV/grcyg26YrSpk2e8ooIrWChpe38ab6ivACFrW1KM7Jz7/79h/ceEJLj9EtqR4sn+tIsp9WUskmnRJoJelpTLuPxseoZkw/fsLjm8t7a9d3XbdauKd10vW0/Aqmpt+1MPoJaICRomf6YKPaKWBJTQwShWQriS/l3aOIoKSDEfUpRDc2SSghPC03F8FusGl9ki/zlLum7GIxztHieTRBPyfek9aiBYnoT7J/MyvXtgO/du7uze3dvZ+9+2lbp7sO7O3v8l+lIfggdzL/yHxmE5VtilJJWrsCezTzPaa2svV4Rp7b5u/YuCb5ZFtq0YEF/mnWDGnmLbXpvTP//CLNA/UD59IdKUqMs/XldzH7/1bI8WTdttSga5k58CL30YwZp+if41zy3A1/nK0Kcf5V21N9rWTCGNiIe4860N/8ZmqqbOqCp7HRgpk8Xqs3HNwo8ECDkCJeNj8/3/sPLPO+PP72l+L+GKshIILsj8L6ILyORlxBZQsrK9rOP6BUwHUkbJaloZVe04M7q3eE8R8KCf5UP70Mz9oZGs2XtJv3Pe8B09MM+4BBqRUvrbTYpSd/JIC5orDSivCyJwFOivyxe0d+rbIaZ+eyje8G61vvT8KKe6Qdvssm6zOqfLPIrO3IZHenU3/1QeqG16bLMVk3+yPxCFgGIE+akcQXtObFNXr+qrvQr+m6eNlOSRcKuIo1AVltk9oalXB3E7xYO4ndzg8BiL7WLDoKWfV9Xdfu7vcoWZzMsANOsG8MCB+Hx3XZ+9HOK2fNqJqsH0J3AkD/QfOjPNXKUp6WcMwXNx7MZ6cwG+NnPUv3w/x2oPs2lB5pYYOn9afFjPqTfasLU8ukSwykDPp39cDDn7/b4310aBbB++RI2kSzlQ0WaUKH/7z5w5pL+xEd7O+nJuiarSD7NqyqbpS+/fPUmPTn+4uWT0+fP0588O0nv7X1KYKgt2hOmTVuvWR+m1Tm5YemX63ayLkqoEG7Gw6ffAvKQEszrJbHmzxmF7vG/PQodhBQi96JLofTb67rMr5U+r3+vsydfGrrs2FZvrqq0rMiYrSekfosG5Nl6lWdlcbGEVIJWEziLWX19h6g2S2lBgn8+zS/zslqZRmQcZ+RW1mmWzq5oWjZS9eeY6fb53x5JH4Qk3euR9GH6PP28zq4xViXryZdPnp6+MHT91DYlJIXfUiLJNKvr67Ratykt9bxtUqJBNS2IqWbpVUFK42w5I5+Kmmy9oVfept/NmvkdBsXUod8C6v3c8+R9/rdHQBq+T8DdHgHvpSek6wn7a6Hem9NXxy8+N9Tbt+1e5YvqkpZdia9ekK2isOkn8wuKYFh6t75c5ulrsmEEhyj4ps7zDcT6OWa1T/nfHqWIJD6ldnqUeniwk36Rl3VF0sakekmL7ienb778rmU2p9w8cp1SpMgiuXVakvdXF9OivSYaZcuGXHaIt0jxSV1dLcHM1JSGlteXTNwm3fqCQt6agNGn1bqe0lK3L+nHxNG7Dx/ubKD4zz17PuB/e0Qn/tpM9L37n6ZfZMuC4nBSmC0xVssm5fTVU9Bd6e6U5zFGqnQDUU/fEeKgMWK8LH2qelCEPEu/Tdm5lrzs9MvLvKb8AsNh+tBvAQl/jpn2gP/t0e9eQL++TX64vXsv/XbVrGiUxD/g2jdnXzz58kvLsmS0qB3awu4879gdkJCQJ024tnYa7ZiOywtytOhFpgf9FtDr557lHvK/PZLtBiTrG+nd++l3KS1DY30t7DakEQnDgC639V9+bjlpdwf/9slCIuSThSRRciP2o093Pk3fkGwtL7a/IO+2zN6qGnzx5Zevjt8Y+rgXTubUNgdl1qQxSfyOp9NqsahmLJ7cjMdPvwX0GeacHxaJdvFvj0Q7pN03k+je/e17D1inQIyEe56fvf7iy1enQp57e459fjKrC6YESPOKrOZz0vLpcZ1n7LO9zijeJtq9pJGRgsLXv2hd1fx6lGw/x2wVjxp2SL14NNujJFuHZrsUs+XL/NrQa5NyJyxF4jy1TiT6SXDlLKM1LmrDY6ffAtr83LNUPGTYCf3bvYMeeQ4epE/IVc1by1CD+tv3CYgq5PRXayRtmmKWz0bpeVlVs5KSZIvsp4m/KCO5qJj9GoorGAjThX4LSPdzzFbxuGAndGsjdNt9SDnBjHwl0VHvGYqacCnduiB5JN6hDD+p94oElBKtNXFrsUwl2/b/Zq9rNx4U7BCZNlNv7+Bh+qReLyYmVH15SlbwyZevvvzq828r+R44uUT8SVRDDMa6i8ZHQ0nPs3pBVnGWXtTVepVmG9Q/fqNfkGTE7zY96T3hX7GU7e3+Qo7TwHf/0Rf0Df3DOU87g8wRT9ZtSwzyPD8PlggfF8sVxY+S9W/WE0pAf5QuswX9Baq7+Zu0y5d1fllUa1oUoKBgTS3cB/3caNCeqEYJrbf5Nb1DSwxY8vvsI7Kk9GH6e+XXj1L6VLEFgsc1RRGCsEv8+sOlP3uPP+QprWzRGpF5hb7tPTHam+frfDfUV+zz6GcrQnfV+9CMqFjCmSXOwqh6zQicTEHTPKmIbAtSt5ai9OGL6mVdbFgb7r77nJYPKdFIX4dP/3U8pAoYAM//fKULObJ+T/7A292AAX4yygD0qaiTWywZfk6mliLJbSwfN7SAWFxm0+ttLBtyiErws5qi+s8++v2bvDwnlSEtoC8GBkCLHd4Q6C9B3i5w0AiUmBRhtZTnLolIlMi/adWDjLpp/aQkpRhbABENTOsAdsFjl37TFQH5AF85Qeg+N5F/j3pkSjP53xB5+uSnT29N/tfZglaOlfpPi4YoUywIV1KS2yfV6rrGeKiXcBJcww+YBxoKAfn/6DzQapI3D18RhfrzQJ/eeh6gaUm3E5yQ0vo5hbzr5bQoQW7CLvYMjIPWw/wVzy+qSVHmGEFMf5inM3dM+P9Xiw2e7pTJUOF5mHm4wbsJ3lEPRT5I8cn7035otTdmMvBE1beTwqNfSOtG14cU807ny6qsLq5TZDmfF2CmWSr+EoHoQI595v7yv4U1fiwEUieizd+1dx3ZwDN37z7+Xb938vT4zfH3fuPkd9uaVdM1FPWdMfmls+utc+JTuCjp1p30Fxfn6dbvtvXxjzN9f39yNE7JGahphWxcNFsfP1Ir+PGd9LPPUvJ2c3oFzX//3//1yasvnz9/+eXrszdnX774vekFennr452P7xzGGvw+foNf8kvuHN69+/3vE640A4w3/0p+4IJ+UVfux+g3eeRv82fwl/+H+d1SjGdLReKcFcM2Ze/p3ylktiCzRq4nf3xe1E27TW4Q6Ed0Dp4IGAeAfZ/z4l3sRTz+y1gDJ8e2mG3TmvfQC3j8l2gNcvtds727l+K3ZvtT/rnY3k/pF8Ji2m6vm23FiySRdBNAEz3wPJ7fM4C0CatBp7a+onzc/J73Aq3GYzmUGlg1mL6eF3VuXDFqhGf3YLebZe20OCHLldczylCPSDynZFCKbERRAKJz04ttDIeMYgDWAr42vqiqizInpbyAEl7QMv3dFSmp/C71/4np/5PX7egT190nFG18gn7u/p7b9w7Ge/f2Dj7duT/a3b833t3f39vbG+0++MFdCiqyz37Xe4vd33V/Qv9f3Kff93/X3Wbn3afZbC/PdycH5wf3p5/ev/9o592D2c5sMnt4MLlHCZnZvXu/68Fi73e9N/PA/677M+ng0/3dnQe/x7z8LCd32hqLSZlBo0NNUUxIlFsREY46FOv8+SYv89UcEeSjdGvn3p30/v2H99IHuzs7nYani6woqZGlIP5uq0cN5u33jBu0j442fcvIsedL4OkxIkW/0rOJP/UXMOht+PD1PKvz9BfS3B2SCV0uabWjw5DrsvMmRdm0+oZs/HZBItCkvAxHXeazIpNv4UxbCD/2uCwiILbPiY0mVUWz4poS/ysJfSY0LcGG5AhYgrFceJMsHoEPrSAqFkfPC0p8rhtMvAHlNyJauz8f3y0L/8847i0Nn/4IO+uiro16WJ94KCtf+mAE52dVWVZXirVC8lt9DaQLyphkF3W2CPvrog2K26aMvMejzLR3vQEM0fzMQPC/+hpIX5PJWE+IU13LOMrakBGm7HXdw/oWSH83a6dzWmKu09+nWr8haCky4su89Jv2B+F4/fHddWm/NUJLFqX3DH1nPrc/6UeHLDRO+pckT02gftzk9NHsh2tEw76z+prnjMQ/DgePDwuKyumuxfaDTS/iwctxO+VNtqdGO8Gzi8q64VuHNVIvgON0mG2K2f+Fy0mzOvwZp+7fGxGN0W+O4smya4tvpuPX9C0Z8V53+vnX7eR4+6e2z5az/F0P8HG6nf5Uyt8BuGFrmsvoc+P39PUQ99xPV9VVXuez7cn1bRjJwOF5vekFPBpd7O3sPiCGsIQQ94w+Ice+KJV88kOCS+3Hx+6l/J5OrkM6K5mrVU6w2iJviOLwvDy6qsX4kpqccBOhK3q6YQw3EnfD90Pfmc/NT/oIj/sAv02q2TX9So5Fuyjpl984+X8A4ifVtgVgAAA=
  recorded_at: Tue, 14 May 2019 14:00:00 GMT
recorded_with: VCR 6.0.0
