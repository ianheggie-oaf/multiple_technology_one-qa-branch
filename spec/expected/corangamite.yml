---
- council_reference: PP2025/005
  address: 289 Brumbys Road PETERBOROUGH VIC 3270
  description: Use of land for a host farm and group accommodation
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2025%2F005&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-01-28'
- council_reference: PP2025/006
  address: 19 Pearce Road PORT CAMPBELL VIC 3269
  description: Construction of dwelling (greater than one storey in height)
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2025%2F006&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-01-28'
- council_reference: PP2025/007
  address: 87 Barrett Street TIMBOON VIC 3268
  description: Development of a double sided, floodlit major promotion sign
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2025%2F007&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-01-28'
- council_reference: PP2025/008
  address: 1 Adeney Street CAMPERDOWN VIC 3260
  description: Construct Extension of Verandah
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2025%2F008&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-01-29'
- council_reference: PP2025/009
  address: 35-37 Heriot Street LISMORE VIC 3324
  description: Variation to Red Line Area for Sale of Packaged Liquor
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2025%2F009&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-01-30'
- council_reference: PP2025/010
  address: 606 Terang-Mortlake Road NOORAT VIC 3265
  description: Change of use to Accommodation
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2025%2F010&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-01-30'
- council_reference: PP2025/011
  address: 15 Wilson Street TERANG VIC 3264
  description: Construction of an Outbuilding
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2025%2F011&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-03'
- council_reference: PP2025/013
  address: 9-13 Hospital Road TIMBOON VIC 3268
  description: Two Lot Subdivision and Construction of Two Dwellngs
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2025%2F013&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-07'
- council_reference: PP2025/014
  address: 256 Manifold Street CAMPERDOWN VIC 3260
  description: Alterations and Extension to a Dwelling with a Heritage Overlay
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2025%2F014&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-10'
- council_reference: PP2025/015
  address: 980 Melrose Road PRINCETOWN VIC 3269
  description: Removal of Easement (Electricity Transmission) and Crown Grant Reservations
    (Mineral Resources Development Act 1990)
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2025%2F015&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-10'
- council_reference: PP2025/016
  address: 3 Cemetery Road TERANG VIC 3264
  description: Removal of Native Vegetation (One Scattered Tree)
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2025%2F016&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-11'
- council_reference: PP2025/017
  address: 9 L Graylands Road COBDEN VIC 3266
  description: Construct or carry out works associated with Industry (Truck Wash)
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2025%2F017&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-12'
- council_reference: PP2025/018
  address: 2 Hurleys Road SKIBO VIC 3260
  description: Two lot Subdivision (Realignment of boundary) and Use and Development
    of land for a dwelling
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2025%2F018&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-13'
- council_reference: PP2025/019
  address: 20 Currells Road PORT CAMPBELL VIC 3269
  description: Construction of an Outbuilding
  info_url: https://eservices.corangamite.vic.gov.au/T1PRprod/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=PP2025%2F019&f=%24P1.ETR.APPDET.VIW&r=CSC.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-17'
